client/app/bundles/course/survey/reducers/utils.js

```diff
@@ -0,0 +1,18 @@
+export const findById = (array, id) => (
+  array.findIndex(item => String(item.id) === String(id))
+);
+
+export const updateOrAppend = (array, item) => {
```

Comment:
Can we document these methods ?  later we could move this to `lib/helpers/array` if other places need it.